stubs/emoji/emoji/core.pyi

```diff
@@ -1,17 +1,38 @@
+from collections.abc import Callable
 from typing import Pattern
 from typing_extensions import Literal
 
 _DEFAULT_DELIMITER: str
 
+class _DeprecatedParameter: ...
+
 def emojize(
     string: str,
-    use_aliases: bool = ...,
+    use_aliases: bool | type[_DeprecatedParameter] = ...,
     delimiters: tuple[str, str] = ...,
     variant: Literal["text_type", "emoji_type", None] = ...,
     language: str = ...,
+    version: float | int | None = ...,
```

Comment:
`int` is redundant with `float` in argument types:

```suggestion
    version: float | None = ...,
```

@AlexWaygood Maybe that's a check we could add to flake8-pyi?

Reply:
Whooops, looks like we did simultaneous reviews! And, yeah, we could, certainly :)